grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -510,6 +513,177 @@ class ApiController extends BaseController {
+                    @ApiResponse(responseCode = "404",
+                            description = "Opus or profile not found"),
+                    @ApiResponse(responseCode = "500",
+                            description = "An unexpected error has occurred while processing your request.")
+            ],
+            parameters = [
+                    @Parameter(name = "opusId",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "Collection id - UUID or short name"),
+                    @Parameter(name = "profileId",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "Profile id - UUID or Scientific name"),
+                    @Parameter(name = "type",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "type - private or open"),
+                    @Parameter(name = "Access-Token",
+                            in = ParameterIn.HEADER,
+                            required = false,
+                            description = "Access token to read private collection"),
+                    @Parameter(name = "Accept-Version",
+                            in = ParameterIn.HEADER,
+                            required = true,
+                            description = "The API version",
+                            schema = @Schema(
+                                    name = "Accept-Version",
+                                    type = "string",
+                                    defaultValue = '1.0',
+                                    allowableValues =  ["1.0"]
+                            )
+                    )
+            ],
+            security = [@SecurityRequirement(name="auth"), @SecurityRequirement(name = "oauth")]
+    )
+    def getLocalImage () {
+        if (!params.type) {
+            badRequest "type is a required parameter"
+        } else {
+            try {
+                ImageType type = params.type as ImageType
+                if (type == ImageType.PRIVATE) {
+                    def result = apiService.displayLocalImage("${grailsApplication.config.image.private.dir}/", params.opusId, params.profileId, params.imageId, false)
+                    if (result) {
+                        response.setHeader("Content-disposition", "attachment;filename=${params.imageId}")
+                        response.setContentType(Utils.getContentType(result))
+                        result.withInputStream { response.outputStream << it }
+                    }
+                }
+            } catch (IllegalArgumentException e) {
+                log.warn(e)
+                badRequest "Invalid image type ${params.type}"
+            }
+        }
+    }
+
+    @GrantAccess
```

Comment:
don't think grant access is needed. all access to private collection image must have valid access token.